tracer/src/Datadog.Trace/Iast/Mark.cs

```diff
@@ -0,0 +1,15 @@
+// <copyright file="Mark.cs" company="Datadog">
+// Unless explicitly stated otherwise all files in this repository are licensed under the Apache 2 License.
+// This product includes software developed at Datadog (https://www.datadoghq.com/). Copyright 2017 Datadog, Inc.
+// </copyright>
+
+using System;
+
+namespace Datadog.Trace.Iast;
+
+[Flags]
+internal enum Mark
```

Comment:
I don't know if defining it as byte will make a difference in mem consumption (because of alignment), but it wouldn't hurt to try :)

Reply:
True :)
Applied in 2338cd6